### DISCUSSION TITLE
Possibility in borg to add redundancy?

### DISCUSSION THREAD

**Discussion by cameo69** [borgbackup/borg#6030] [Q&A] [answered]
I searched the discussions but could not find any hint if there is a way to add some redundancy to borg-archives to have resilliance against data corruption. I am thinking about a way to add some extra data to increase the chance that my backup can be extracted even if over time some bits flipped.
I know that [DAR](http://dar.linux.free.fr) offers a feature ([Parchive](http://dar.linux.free.fr/doc/usage_notes.html#Parchive))to use a par2 installation to add redundancy.
Has borg any similar concept?

**Comment by ThomasWaldmann** [comment, marked as answer]
I think we have at least 1 ticket about this in the issue tracker, just search for ecc, par2, redundancy.

borg does not offer anything related to this.

An easy and recommended way to add redundancy is of course to just have multiple backups of the same data in different repositories at preferably different locations.

**Reply by cameo69** [reply]
Thank you, how could I miss this? --> e.g. https://github.com/borgbackup/borg/issues/225
